pkgs/top-level/all-packages.nix

```diff
@@ -12847,11 +12847,10 @@ with pkgs;
     emacs30-nox
     emacs30-pgtk
 
-    emacs28-macport
-    emacs29-macport
```

Comment:
Aliases and release notes have to be added for deleted packages.

- doc
  - https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#release-notes
  - https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#deprecatingremoving-packages
- example: #386593

Reply:
I added aliases, but I'm not sure exactly where to add it to release notes.